ChaosMod/Util/Vehicle.h

```diff
@@ -28,4 +28,30 @@ inline Vehicle CreateTempVehicleOnPlayerPos(Hash model, float heading)
 	SET_VEHICLE_AS_NO_LONGER_NEEDED(&dummy);
 
 	return veh;
-}
\ No newline at end of file
+}
+
+inline std::vector<Vehicle> GetNearbyVehicles(Ped aroundPed, float radius, int maxAmount)
+{
+	Vector3 position = GET_ENTITY_COORDS(aroundPed, false);
+	int amount = maxAmount;
+	int* handles = new int[amount * 2 + 2];
+	handles[0] = amount;
+
+	std::vector<Vehicle> foundVehs;
+	int foundAmount = GET_PED_NEARBY_VEHICLES(aroundPed, handles);
+	for (int i = 0; i < foundAmount; i++)
+	{
+		int index = i * 2 + 2;
+		if (handles[index] != 0 && DOES_ENTITY_EXIST(handles[index]))
+		{
+			Vehicle veh = handles[index];
+			Vector3 vehPos = GET_ENTITY_COORDS(veh, false);
+			if (vehPos.distanceTo(position) < radius)
+			{
+				foundVehs.push_back(veh);
+			}
+		}
+	}
+	delete[] handles;
+	return foundVehs;
+}
```

Comment:
Same applies here, no need to execute a native for that when you have a `GetAllVehs` helper function already